### ISSUE TITLE
Add Greedy Algorithms

### ISSUE THREAD

**Issue by abhisheknalla** [khanchi97/Smvdu-Algos#132]
Sir, I would like to add the follwoing Greedy Algorithms

Standard Greedy Algorithms :

    Activity Selection Problem
    Egyptian Fraction
    Job Sequencing Problem
    Huffman Coding
    Efficient Huffman Coding for sorted input

Greedy Algorithms in Graphs :

    Kruskal’s Minimum Spanning Tree
    Prim’s Minimum Spanning Tree
    Boruvka’s Minimum Spanning Tree
    Reverse delete algorithm for MST
    Dijkastra’s Shortest Path Algorithm
    Dial’s Algorithm
    Dijkstra’s Algorithm for Adjacency List Representation
    Prim’s MST for adjacency list representation

Can I start working on this ?

**Comment by khanchi97** [comment]
Yes you can start working on this.Just make sure that you don't code such a implementation which is already implemented :)

**Comment by akarshsingh9** [comment]
@abhisheknalla  can i work along with you on this ?
